config/resources/submissions-domain.lisp

```diff
@@ -112,3 +114,19 @@
   :resource-base (s-url "http://themis.vlaanderen.be/id/voorlopig-bestand/")
   :features `(no-pagination-defaults include-uri)
   :on-path "draft-files")
+
+(define-resource submission-internal-review ()
+  :class (s-prefix "subm:InterneBeoordeling")
+  :properties `((:created           :datetime ,(s-prefix "dct:created"))
+                (:modified          :datetime ,(s-prefix "ext:modified"))
+                (:private-comment   :string   ,(s-prefix "ext:privateComment")))
+  :has-one `((subcase               :via      ,(s-prefix "subm:isInterneBeoordelingVoor") ;; reuse the same when postponed/retracted and reproposed?
+                                    :as "subcase")
+             (user                  :via      ,(s-prefix "ext:modifiedBy")
+                                    :as "modified-by"))
+  :has-many `((submission           :via      ,(s-prefix "subm:heeftInterneBeoordeling") ;; for BIS updates, can we just look at the subcase at that point?
```

Comment:
> for BIS updates, can we just look at the subcase at that point?

I think it's easiest for the frontend logic to just use the same URL as the existing subcase, but indeed, this will be the same for BIS updates.

Reply:
I put down the comment initially to make a decision on one-to-one or one-to-many.
We could connect the review model to 1 submission (the initial one) and connect it to subcase later.
Then use that subcase.internalReview model on BIS updates via subcase and don't connect/get it to the BIS submission.
But opted to go for one-to-many first. so internalReviews are linked to the subcase and all the submissions of that subcase

Reply:
both have benefits,
one-to-one does mean not all submissions will have a internalReview, which may be confusing.
maintaining a one-to-many with concurrency can be more difficult. We have to check if the list we have is stale and update before saving the model to prevent losing connections to submissions.